site/contents/write_ups/2021/03-11-21_writeup.md

```diff
@@ -0,0 +1,109 @@
+
+You will not have innovation in the long term if people do not trust your product - they will just pull the plug. A lot of the AI being commerical ties into what public thinks about it. If people don't like a product, they won't use it. Public opinion almost forms an accidental ethics board in a more consumerist, capitalist society. 
+
+Consumerism should not be the only line of defense to unethical practise. We need formal ethical boards to block unethical developments as well as guide innovators. Scientists should innovate in tramlines rather than in an open space.
+
+Facebook appear to have reached a point of innovating too hard and not building in enough data protections where we need. In order for people to trust that you're going to innovate things, you need to show you're going to say no sometimes and that there is a line in the sand. Journals seem to not have as many moral standards and are more focussed on "quality" of science and innovation. They also need to do better.
+
+Google had an Ethical AI team, but [they got fired](https://www.bbc.co.uk/news/technology-56135817) when they spoke up about unethical practises. We cannot rely on tech companies to regulate themsleves as we have seen previously in discussions on 
+
+There aren't typically headlines about ethics reviews being rejected. Often there aren't reviews to pass through, but a journal shouldn't tweet everytime they reject a paper. But perhaps highlighting "hey this kind of paper is not entirely ethical" would be a worthwhile. These usually take the form of twitter feeds expressing outrage at certain developments, but not formal objections. 
+
+There is an interesting analogy with pharmaceuticals and drug testing. There are stringent requirements in the pharmaceutical industries enforced by government agencies such as the [MHRA](https://www.gov.uk/government/organisations/medicines-and-healthcare-products-regulatory-agency) in the UK and the [FDA](https://www.fda.gov/) in the US. Can we get something like this for AI?
+
+What would the phases of AI trials look like? It would probably depend on what sort of system you were testing, just as there are different [clinical trials](https://en.wikipedia.org/wiki/Clinical_trial#Trials_of_drugs) for drugs, procedures and devices. the system (unlike drugs) throughout that process. Would it involve some sort of [Randomised Controlled Trial](https://en.wikipedia.org/wiki/Randomized_controlled_trial)?
+
+Whilst this is a nice idea in theory, private companies run the game with AI and it's unlikely a government institution will be able to dislodge half of Apple or Google, especially against the uphill battle of lobbying. This is the tradeoff of the consumerism focus of society. Consumers have a small sway in the directions these companies go by favouring certain products, but the more we support and consume AI developments, the more purview we give these tech giants.
```

Comment:
```suggestion
Whilst this is a nice idea in theory, private companies run the game with AI and it's unlikely a government institution will be able to dislodge half of Apple or Google, especially against the uphill battle of lobbying. This is the tradeoff of the consumerism focus of society. Consumers have a small sway in the directions these companies go by favouring certain products, but the more we support and consume AI developments, the more purview we give these tech giants. We also worried that Big Tech has too much political influence at this stage for any country to be successful in implementing regulations that would severely damage their income; the only way this could really happen would be to go back in time and implement regulations before the concerning AI practices had started to be used by huge companies like Meta and Google! Alas, unlikely. 
```